messaging/src/main/java/org/axonframework/serialization/ContentTypeConverter.java

```diff
@@ -16,38 +16,42 @@
 
 package org.axonframework.serialization;
 
+import jakarta.annotation.Nonnull;
+import jakarta.annotation.Nullable;
+
 /**
- * Interface describing a mechanism that converts the data type of IntermediateRepresentations of SerializedObjects for
- * Upcasters. Different upcasters may require different data type (e.g. {@code byte[]} or
- * {@code InputStream}), or may produce a different data type than they consume.
+ * Interface describing a mechanism that converts the data of a specified {@link #expectedSourceType() source type} into
```

Comment:
I technically think w don't convert the data, but the object, right? So 

```suggestion
 * Interface describing a mechanism that converts an object from a specified {@link #expectedSourceType() source type} to
 ```

Reply:
Yep, you're totally right here. I'll take your suggestions right away.